Components/Terrain/include/OgreTerrain.i

```diff
@@ -0,0 +1,47 @@
+%module(package="Ogre") Terrain
+%{
+/* Includes the header in the wrapper code */
+#include "Ogre.h"
+#include "OgreTerrain.h"
+#include "OgreTerrainAutoUpdateLod.h"
+#include "OgreTerrainGroup.h"
+#include "OgreTerrainLayerBlendMap.h"
+//#include "OgreTerrainPagedWorldSection.h"
+//#include "OgrePageManager.h"
+//#include "OgrePagedWorld.h"
+//#include "OgreTerrainPaging.h"
+%}
+
+%include std_string.i
+%include exception.i 
+%import "Ogre.i"
+
+#define _OgreTerrainExport
```

Comment:
```suggestion
#define _OgreTerrainExport
#define OGRE_FORCE_INLINE
```
should be here

Reply:
Had the same issue 🙁: 
```
Macro 'OGRE_FORCE_INLINE' redefined.
```

The previous definition is at `OgrePlataform.h`. Somehow the file I have is different from the one on this repository. But here's how it's in mine
```
/* See if we can use __forceinline or if we need to use __inline instead */
#if OGRE_COMPILER_MIN_VERSION(OGRE_COMPILER_MSVC, 1200)
    #define OGRE_FORCE_INLINE __forceinline
#elif OGRE_COMPILER_MIN_VERSION(OGRE_COMPILER_GNUC, 340)
    #define OGRE_FORCE_INLINE inline __attribute__((always_inline))
#else
        #define OGRE_FORCE_INLINE __inline   <----------------------- This is being defined
#endif
```